src/index.js

```diff
@@ -84,4 +84,13 @@ Element.listBody.addEventListener('change', (e) => {
     newTask.changeComplete(Number(e.target.ariaLabel));
     refresh();
   }
-});
\ No newline at end of file
+});
+
+Element.clear.addEventListener('click', () => {
+  const task = newTask.listArray;
+  task.forEach((item) => {
+    if (item.completed === true) {
+      newTask.clearCompleted(item);
+    }
+  });
+});
```

Comment:
**[OPTIONAL]**

- When a user clears a completed task, it gets cleared in the local storage but the UI is not updated. Please, ensure that the UI gets updated when the user clicks on the clear completed button rather than reloading the pages over and over again this will improve the user experience.